source/s3_auto_ranged_put.c

```diff
@@ -557,176 +453,159 @@ static int s_s3_auto_ranged_put_prepare_request(
     return AWS_OP_ERR;
 }
 
-static int s_s3_auto_ranged_put_header_block_done(
```

Comment:
Similar to auto-ranged-get, we no longer spread response response handling over a series of callbacks.  Anything that was in those callbacks, is meant to be isolated in the request_finished function.